### PR TITLE
Python formatting & annotation of Kleinstiver mutants. See README upd…

### PR DESCRIPTION
**Pull request by alexanian**
…ate for details.

Specifically:

kleinstiver_csv_to_dict.py parses the CSVs into dictionaries which are annotated using the original PI domain sequence (stored in cas9_mutants.py as PI_domain), the amino acid group (stored in cas9_mutants.py as aa_group) and the secondary structure region where the mutant occurs (stored in cas9_mutants.py as PI_sec_structure). The output of that script was then copied into cas9_mutants.py as kleinstiver_mutants.

So basically the annotated kleinstiver mutants are now available in pythonic form. Advice on how to more intelligently format the data is welcome.

### CODE REVIEW

**Comment by mattsmart** [comment]
I'd suggest compressing the dictionaries, maybe like this
redo1
I can do it after the merge if you like, its pretty boring work

**Comment by alexanian** [comment]
It's not that boring b/c just a change to the kleinstiver_csv_to_dict.py script. I think I did what you were hoping for?

File: models/tridimensional/cas9_mutants.py [hunk 1]
```diff
@@ -0,0 +1,800 @@
...
+                     'aa_group_init': 'hydrophobic', 'aa_group_mut': 'basic'},
+                    {'aa_idx': 1218, 'sec_structure': 'beta_strand_14', 'aa_init': 'G', 'aa_mut': 'R',
+                     'aa_group_init': 'special', 'aa_group_mut': 'basic'},
+                    {'aa_idx': 1297, 'sec_structure': 'turn_7', 'aa_init': 'H', 'aa_mut': 'D',
+                     'aa_group_init': 'basic', 'aa_group_mut': 'acidic'},
+                    {'aa_idx': 1335, 'sec_structure': 'none', 'aa_init': 'R', 'aa_mut': 'T',
+                     'aa_group_init': 'basic', 'aa_group_mut': 'polar'},
+                    {'aa_idx': 1337, 'sec_structure': 'none', 'aa_init': 'T', 'aa_mut': 'R',
+                     'aa_group_init': 'polar', 'aa_group_mut': 'basic'}]}
+]
```
Comment (mattsmart) [inline models/tridimensional/cas9_mutants.py]: EOF newline

Comment (mattsmart) [inline models/tridimensional/cas9_mutants.py]: can we also move the list close to the end of the last dictionary entry? I think its clear enough that its a really long list lol

File: models/tridimensional/kleinstiver_csv_to_dict.py [hunk 2]
```diff
@@ -0,0 +1,65 @@
+"""
+Convert CSV of Cas9 mutants from Klienstiver et al. (2015), Nature, doi:10.1038/nature14592 into dict format
+Included for reproducibility of data preparation
+"""
+
+from cas9_mutants import PI_domain, PI_sec_structure, aa_group
+import os
+import csv
+
```
Comment (mattsmart) [inline models/tridimensional/kleinstiver_csv_to_dict.py:7]: minor stuff on import order convention: usually all external libs are imported first, alphabetically (the ones starting with import ..., then the ones starting with from ... import blah, then the local libs are imported the same way

pep guide: Imports should be grouped in the following order:
standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.

`import csv`
`import os`

`from cas9_mutants import PI_domain, PI_sec_structure, aa_group`

I also think its more common to put the imports at the very top, even above top-level comments

Comment (alexanian) [inline models/tridimensional/kleinstiver_csv_to_dict.py:7]: "Imports are always put at the top of the file, just after any module comments and docstrings, and before module globals and constants." - https://www.python.org/dev/peps/pep-0008/

So I think comments stay above, but will reorder the import statements.

**Comment by mattsmart** [comment]
Looks good!
